[changelog] Add AAC encoder bugfix to changelog (#722)
software-mansion/live-compositor: CHANGELOG.md

### ✨ New features

- Add `loop` option for MP4 input. ([#699](https://github.com/software-mansion/live-compositor/pull/699)) by [@WojciechBarczynski](https://github.com/WojciechBarczynski)

### 🐛 Bug fixes

> - Fix AAC output unregistering for empty audio streams. ([#714](https://github.com/software-mansion/live-compositor/pull/714)) by [@WojciechBarczynski](https://github.com/WojciechBarczynski)

Reviewer: ```suggestion
- Fix AAC output unregister before the first sample. ([#714](https://github.com/software-mansion/live-compositor/pull/714)) by [@WojciechBarczynski](https://github.com/WojciechBarczynski)
```
or
```suggestion
- Fix AAC output unregister request before the first sample. ([#714](https://github.com/software-mansion/live-compositor/pull/714)) by [@WojciechBarczynski](https://github.com/WojciechBarczynski)
```
Not sure if it is just me, but unregistering sounds a bit weird
